client/components/themes-list/index.jsx

```diff
@@ -91,13 +105,39 @@ export const ThemesList = React.createClass( {
 				/>;
 	},
 
+	handleUploadThemeClick() {
+		this.props.themeUploadClickRecorder(); // tracking
+		this.props.onThemeUpload();            // redirect
+	},
+
+	renderThemeUploadBox() {
+		this.props.themeUploadShowRecorder();
```

Comment:
I don't think that this is required. We can track the number or users that have seen this variant with `abtest( 'signupThemeUpload' )`, since the tracks events for test variants are recorded implicitly.